lib/needlessDisables.js

```diff
@@ -90,7 +90,8 @@ module.exports = function(
 				// mark this range as unused
 				if (!range.used && !alreadyMarkedUnused) {
 					unused.ranges.push({
-						..._.omit(range, 'strictStart', 'strictEnd'),
+						start: range.start,
+						end: range.end,
```

Comment:
How can we be sure, that we don't miss any property here? Maybe there are more properties.

Reply:
Actually I got it from types, but there are tests on this https://github.com/stylelint/stylelint/blob/master/lib/__tests__/needlessDisables.test.js#L40